### DISCUSSION TITLE
Verified Camera

### DISCUSSION THREAD

**Discussion by leosperry** [roleoroleo/yi-hack-Allwinner#417] [General]
I've installed the y20ga image onto a camera with the following stats 

- Firmware Version: 0.3.6
- Base Version: 12.2.0.2_202304071306
- serial number: 0000SFUSY318Q9N5UI23

At a minimum, the rtsp stream is working and all other features appear to be working as well.

**Comment by leosperry** [comment]
The text to speech is not working, but I'm not disappointed. It would've been a bonus.

**Reply by roleoroleo** [reply]
Thank you for your feedback.
About tts, did you install the engine?

**Reply by leosperry** [reply]
Perhaps I missed something. Do you have documentation on this engine?

**Reply by roleoroleo** [reply]
https://github.com/roleoroleo/yi-hack-utils
